source/installation/index.html

```diff
@@ -256,87 +256,62 @@ <h3>Install Home Assistant on x86-64 machines</h3>
 
   <h2>Advanced installation methods</h2>
   <p>
-    Home Assistant offers four different installation methods. We recommend
-    using Home Assistant Operating System. Other methods are available for
-    experienced users for their specific needs, for example, running Home
-    Assistant in a virtualized environment (e.g. Proxmox), or on top of an
-    existing operating system (e.g. Windows, macOS, Linux):
+    Home Assistant offers two different installation methods:
   </p>
   <ul>
+    <li><b>Home Assistant Operating System</b>: An embedded,
+    minimalistic operating system designed to run the Home Assistant ecosystem
+    on single board computers (like the Home Assistant Green or a Raspberry Pi) or Virtual Machines. It
+    is the most convenient option in terms of installation and maintenance and it supports
+    {% term "add-ons" %}. Home Assistant Operating System is
+    the recommended installation method for most users.</li>
     <li>
       <b>Home Assistant Container:</b> Standalone container-based installation
-      of Home Assistant Core (e.g. Docker).
-    </li>
-    <li>
-      <b>Home Assistant Supervised:</b> Manual installation of the Supervisor.
-    </li>
-    <li>
-      <b>Home Assistant Core:</b> Manual installation using Python virtual
-      environment.
+      of {% term "Home Assistant Core" %} (e.g. Docker).
```

Comment:
I don't think we can mention this as a term anymore, because that was possible when it was an installation method 😄

Reply:
```suggestion
      of Home Assistant (e.g. Docker).
```